Gifski/util.swift

```diff
@@ -2449,7 +2449,6 @@ extension NSViewController {
 		}, completionHandler: {
 			window.contentViewController = viewController
 			viewController.view.animator().alphaValue = 1.0
-			window.makeFirstResponder(viewController)
```

Comment:
@sunshinejr This was the culprit. Why did you add it?

Reply:
Really interesting. So when I pushed the VC it wouldn't be the first responder so I had to do it manually.